tools/download-wheels.py

```diff
@@ -52,6 +52,8 @@ def download_wheels(version, wheelhouse):
         Directory in which to download the wheels.
 
     """
+    if not os.path.isdir(wheelhouse):
+        os.makedirs(wheelhouse)
```

Comment:
```suggestion
    os.makedirs(wheelhouse, exist_ok=True)
```
I think this behaves like `mkdir -p`

Reply:
yes that looks better, thank you!